semantic_segmentation_node:
  ros__parameters:
    input_topic: /camera/left/image_color
    publish_topic: /camera/left/segmentations
    model_path: /perception_models/semantic_segmentation/pp_liteseg_infer_model/
    save_dir: /tmp
    save_images: false
